src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+/**
+ * Deals with making sense of the user command.
+ */
+public class Parser {
+    /**
+     * Parses user input into the corresponding command.
+     *
+     * @param taskString user input
+     * @return corresponding command (enum)
+     */
+    public Command parseStringToCommand(String... taskString) {
```

Comment:
Perhaps declaring what taskString[0] is could make it clearer for readers what it is being compared to.